instrument/build.go

```diff
@@ -59,10 +65,11 @@ var (
 
 // LogBuildInfo logs the build information to the provided logger.
 func LogBuildInfo() {
-	log.Printf("Go Runtime version: %s\n", goVersion)
-	log.Printf("Build Revision:     %s\n", Revision)
-	log.Printf("Build Branch:       %s\n", Branch)
-	log.Printf("Build Date:         %s\n", BuildDate)
+	log.Printf("Go Runtime version:  %s\n", goVersion)
+	log.Printf("Build Revision:      %s\n", Revision)
+	log.Printf("Build Branch:        %s\n", Branch)
+	log.Printf("Build Date:          %s\n", BuildDate)
+	log.Printf("Build DateUnixNanos: %s\n", BuildDateUnixNanos)
```

Comment:
nit: use `%d` instead of `%s` for int64s.

Reply:
sure thing

Reply:
fixed